chia/plotting/util.py

```diff
@@ -233,19 +249,68 @@ def get_filenames(directory: Path, recursive: bool, follow_links: bool) -> list[
         if follow_links and recursive:
             import glob
 
-            files = glob.glob(str(directory / "**" / "*.plot"), recursive=True)
-            for file in files:
-                filepath = Path(file).resolve()
-                if filepath.is_file() and not filepath.name.startswith("._"):
-                    all_files.append(filepath)
+            try:
+                files = glob.glob(str(directory / "**" / "*.plot"), recursive=True)
+                for file in files:
+                    try:
+                        filepath = Path(file).resolve()
+                        if filepath.is_file() and not filepath.name.startswith("._"):
+                            all_files.append(filepath)
+                    except Exception as e:
+                        # If we can't process a specific file, log and continue
+                        log.warning(f"Error processing file {file}: {e}")
+                        continue
+            except Exception as e:
```

Comment:
it seems unnecessary to keep the `glob()` path. The idea is that the "manual" traversal works too, right? It's much easier to test if there's only one implementation